front/src/modules/trainschedule/components/Timetable/PacedTrain/hooks/useOccurrenceActions.ts

```diff
@@ -147,10 +143,29 @@ const useOccurrenceActions = ({
     [pacedTrain, occurrences, selectedTrainId]
   );
 
+  const deleteAddedException = useCallback(async (occurrenceId: OccurrenceId) => {
+    const key = extractExceptionIdFromOccurrenceId(occurrenceId);
+    const newExceptions = pacedTrain.exceptions.filter((ex) => ex.key !== key);
+    const updatedPacedTrainPayload = {
+      ...formatPacedTrainWithDetailsToPacedTrainPayload(pacedTrain),
+      exceptions: newExceptions,
+    };
+
+    storePacedTrain(
+      pacedTrain.id,
+      updatedPacedTrainPayload,
+      timetableId!,
+      dispatch,
+      upsertTimetableItems,
+      removePacedTrains
+    );
+  }, []);
```

Comment:
We probably should add this in the depths
```suggestion
  }, [pacedTrain.exceptions]);
```

Reply:
done !